### DISCUSSION TITLE
Handling BackPressure of Rest Requests

### DISCUSSION THREAD

**Discussion by jaiprasad266** [quarkusio/quarkus#26335] [Q&A]
Hi ,

We have a quarkus app which expose a rest endpoint to fetch data from cosmos db.
Below is how the endpoint looks.

The dao layer executes the sql in a reactive style and returns a Multi<String>.

How to handle a backpressure when there is burst of incoming rest requests in this scenario ?

@Path("/getData")
    @GET
    @Produces(MediaType.APPLICATION_JSON)
    @Consumes(MediaType.APPLICATION_JSON)
    public Multi<String> getData(RequestBody request) throws IOException {
        Multi<String> data = dataService.findData(request);
        return data;
    }

**Comment by aromalmr** [comment]
I have the same question. I went through this url. But still not yet got a solution. 
https://github.com/quarkusio/quarkus/discussions/26335
